shared/reducers/login.js

```diff
@@ -45,7 +45,8 @@ type LoginState = {
     name: string,
     id: string,
     type: 'mobile' | 'laptop'
```

Comment:
add a todo here to use device type. `laptop` doesn't exist anywhere (I don't think)